pkg/ibmvpc/vpc.go

```diff
@@ -70,18 +70,23 @@ type PrivateIP struct {
 	vpcmodel.VPCResource
 	vpcmodel.InternalNode
 	loadBalancer *LoadBalancer
-	// Since not all the LB balancer has a private IP, we create fake Private IPs at the subnets that do not have one.
-	// original - does the private IP was originally at the config file, or it is a fake one
+	// Since not all the LB has a private IP, we create a potential Private IP at the LB's subnets that do not have one.
```

Comment:
Again, now you are presenting the entire relevant cidr, right?
This should be reflected in the documentation

Reply:
there is no change in the implementation, when creating a fake IP, we allocate for it one address, we just made an adjustment in the report - instead of printing the allocated address, we print the block